### PR TITLE
Cleanup after CLI create, fetch, generate and scaffold commands implemented AEA-491

### PR DESCRIPTION
**Pull request by panasevychol**
## Proposed changes

Cleanup after CLI create, fetch, generate and scaffold commands implemented
#930 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/create.py [hunk 1]
```diff
@@ -142,7 +108,7 @@ def create(click_context, agent_name, author, local):
             registry_path=os.path.join("..", DEFAULT_REGISTRY_PATH),
             description="",
         )
-        agent_config.default_connection = DEFAULT_CONNECTION
+        agent_config.default_connection = DEFAULT_CONNECTION  # type: ignore
```
Comment (DavidMinarsch) [inline aea/cli/create.py:111]: why is type ignore needed?

File: aea/cli/fetch.py [hunk 2]
```diff
@@ -66,8 +67,9 @@ def _is_version_correct(ctx: Context, agent_public_id: PublicId) -> bool:
     return ctx.agent_config.version == agent_public_id.version
 
 
+@clean_after
 def _fetch_agent_locally(
-    ctx: Context, public_id: PublicId, click_context, alias: Optional[str] = None
+    click_context, public_id: PublicId, alias: Optional[str] = None
 ) -> None:
     """
     Fetch Agent from local packages.
```
Comment (DavidMinarsch) [inline aea/cli/fetch.py:75]: please update docstrings

File: aea/cli/create.py (hunk 1 above)
Comment (panasevychol) [reply aea/cli/create.py:111]: It’s really weird. I din’t change anything in this line actually and even checked with debugger type of both expression and var and it’s correct. But static type check complains that it’s str, not PublicId.

File: aea/cli/fetch.py (hunk 2 above)
Comment (panasevychol) [reply aea/cli/fetch.py:75]: Done.

**Pull request merged**